src/components/Events/EventsList.vue

```diff
@@ -24,6 +14,16 @@
             class='pagination__btn'
             v-on:click='incrementCurrentPage'>Next</button>
         </div>
+        <div v-if="!noEvents" class="events-container">
```

Comment:
why did you change this from the v-else?

Reply:
Bill could originally make this v-else because he swapped around where the pagination buttons were. He took them from the top of the component and put them on the bottom, meaning now the "events-container" came directly after the span with v-if, so v-else could work. But this isn't how it's laid out on figma. On figma the buttons are at the top of the list of events, and so the div that contains those buttons separates the "events-container" div from the span with v-if, and v-else no longer works.